lte/gateway/docker/.env

```diff
@@ -12,8 +12,9 @@
 COMPOSE_PROJECT_NAME=agw
 DOCKER_USERNAME=
 DOCKER_PASSWORD=
-DOCKER_REGISTRY=
-IMAGE_VERSION=latest
+DOCKER_REGISTRY=docker.artifactory.magmacore.org/
+IMAGE_VERSION=1.8.0
```

Comment:
If I understand that correctly, from now on if you call `docker-compose build` the resulting artifacts will be tagged like `docker.artifactory.magmacore.org/agw_gateway_c:1.8.0` locally, instead of `agw_gateway_c:latest`. Is that desired?

Reply:
That is not desired for the build. I will revert this change and update the installation script for adding the docker registry.